Formula/rethinkdb.rb

```diff
@@ -29,6 +28,10 @@ def install
     # but brew's protobuf is sometimes linked against libstdc++
     args += ["--fetch", "protobuf"]
 
+    # also, allow fetching of boost, since 1.72.3 or above will
+    # not contain the required endian header file
+    args += ["--fetch", "boost"]
```

Comment:
I don't think that we want to vendor boost here. I think upstream should just update the header file to fix the deprecation in 1.73.0

Reply:
If you mean rethinkdb by upstream, then I'm hesitant, since we already did the release of 2.4.1 with boost 1.60.0. I really wouldn't like to do a 2.4.2 just because we would like to update the rethinkdb version on homebrew as well. If you mean boost as the upstream, well, I think there would be more work on rethinkdb's side to use the latest boost version. Also, my previous statement about 2.4.2 release would be true in this scenario as well.

I just found that boost has the 1.60.0 version on homebrew, so we could use `depends_on: boost@1.60.0` and in that case we would avoid fetching boost.

Reply:
This is what I did in c1e15a2